Tools/DumpRenderTree/TestRunner.cpp

```diff
@@ -268,8 +268,11 @@ static JSValueRef removeAllVisitedLinksCallback(JSContextRef context, JSObjectRe
 
 static JSValueRef removeAllCookiesCallback(JSContextRef context, JSObjectRef function, JSObjectRef thisObject, size_t argumentCount, const JSValueRef arguments[], JSValueRef* exception)
 {
+    if (argumentCount < 1)
+        return JSValueMakeUndefined(context);
```

Comment:
I think we’d want to throw an exception in this case rather than silently doing nothing, but I understand this is just test machinery so maybe this is OK.

Reply:
I agree this isn't great. However, this is the pattern in this file. Every function just returns early with undefined if they don't get enough parameters. I couldn't even find one that throws an exception.